packages/workflow-core/src/lib/plugins/common-plugin/vendor-consts.ts

```diff
@@ -6,14 +6,81 @@ export const SANCTIONS_SCREENING_VENDOR = {
   'asia-verify': 'asia-verify',
 } as const;
 
+export const BALLERINE_API_PLUGINS = {
+  ...SANCTIONS_SCREENING_VENDOR,
+  'company-sanctions': 'company-sanctions',
+  ubo: 'ubo',
+  kyb: 'kyb',
+  'resubmission-email': 'resubmission-email',
+  'session-email': 'session-email',
+  'invitation-email': 'invitation-email',
+} as const satisfies Record<string, string>;
+
+export type ApiBallerinePlugins =
+  (typeof BALLERINE_API_PLUGINS)[keyof typeof BALLERINE_API_PLUGINS];
+
+export const BALLERINE_API_PLUGINS_KINDS = Object.values(BALLERINE_API_PLUGINS);
+
 export type SancsionsScreeningVendors =
   (typeof SANCTIONS_SCREENING_VENDOR)[keyof typeof SANCTIONS_SCREENING_VENDOR];
 
+export type ApiBallerinePlugin = {
+  url: string;
+  displayName?: string;
+  method: 'POST' | 'GET' | 'PUT' | 'PATCH' | 'DELETE';
+  headers: HeadersInit;
+  persistResponseDestination?: string;
+  request: SerializableValidatableTransformer;
+  response: SerializableValidatableTransformer;
+};
+
+type ApiPluginOptionsBase = {
+  pluginKind: ApiBallerinePlugins;
+};
+
+type DowJonesOptions = {
+  pluginKind: 'dow-jones';
+  takeEntityDetailFromKyc: boolean;
+  successAction: string;
+};
+
+type ComplyAdvantageOptions = {
+  pluginKind: 'comply-advantage';
+  takeEntityDetailFromKyc: boolean;
+  successAction: string;
+};
+
+type AsiaVerifyOptions = {
+  pluginKind: 'asia-verify';
+  successAction: string;
+  takeEntityDetailFromKyc: boolean;
+};
+
+type CompanySanctionsOptions = {
+  pluginKind: 'company-sanctions';
+};
+
+type UboOptions = {
+  pluginKind: 'ubo';
+};
+
+type KybOptions = {
+  pluginKind: 'kyb';
+};
+
+type ApiPluginOptions = ApiPluginOptionsBase &
+  DowJonesOptions &
+  ComplyAdvantageOptions &
+  AsiaVerifyOptions &
+  CompanySanctionsOptions &
+  UboOptions &
+  KybOptions;
+
 export const SANCSIONS_SCREENING_BASE_RESPONSE = {
   transform: [
     {
       mapping:
-        "merge({ name: 'company_sanctions', status: reason == 'NOT_IMPLEMENTED' && 'CANCELED' || error != `null` && 'ERROR' || 'SUCCESS' }, @)",
+        "merge({ name: 'sanctions_screening', status: reason == 'NOT_IMPLEMENTED' && 'CANCELED' || error != `null` && 'ERROR' || 'IN_PROGRESS' }, @)",
```

Comment:
Lets talk about this change online

Reply:
it was mistake its for sanctions screening